prisma/schema.prisma

```diff
@@ -67,12 +67,19 @@ type product {
   name     String
   quantity Int
   status   status
+  category category?
```

Comment:
Acredito que temos que fazer a categoria um parametro obrigatório. Ou combinarmos entre si de quando for null tratar como se fosse FOOD.

Reply:
Também concordo que deve ser obrigatório

Reply:
@gfmanica @ernonemacher ela ficou dessa forma para não impactar o que já está no banco, não estava aceitando de outra forma porque o que já existe no banco não tem esse campo, logo precisa ser null.
Podemos ter no Front algum tipo de validação que verifica isso?

Reply:
talvez poderíamos inserir tudo hoje como food pra evitar esse problema, não lembro se tem algum produto como bebida, mas dai ajustamos dps qualquer coisa

Reply:
Já alterei no banco e ajustei no código.